Bonsai.Harp.CF/SyringePumpCommand.cs

```diff
@@ -122,49 +120,47 @@ protected override Expression BuildSelector(Expression expression)
             }
         }
 
-        static HarpMessage ProcessEnableMotorDriver() => HarpCommand.WriteByte(address: 32, 1);
-        static HarpMessage ProcessDisableMotorDriver() => HarpCommand.WriteByte(address: 32, 0);
-        static HarpMessage ProcessStartProtocol() => HarpCommand.WriteByte(address: 33, 1);
-        static HarpMessage ProcessStopProtocol() => HarpCommand.WriteByte(address: 33, 0);
+        static HarpMessage ProcessSetMotorDriver(bool input) => HarpCommand.WriteByte(32, (byte) (input ? 1 : 0));
```

Comment:
I would leave the named argument `address` for readability and also remove the space after the cast for consistency with the rest of the code base.